wire-protoc-compatibility-tests/src/test/java/com/squareup/wire/Proto3WireProtocCompatibilityTests.kt

```diff
@@ -573,6 +574,33 @@ class Proto3WireProtocCompatibilityTests {
     assertThat(parsed).isEqualTo(value)
   }
 
+  @Test fun validateMapTypesWithScalarKeyValueDeserializeOmittedKeyValue() {
+    val bytes = listOf(
```

Comment:
🤔 Hm. Actually, I think it would be best to comment why bytes are handcrafted here.

Reason: like noted in the related issue: both `protoc` & `Wire` encodes empty string in map explicity having a value field. So can't do a roundrobin test like encoding with `protoc` and decoding with `Wire` but need to have the bytes like allowed by standard (and done by ScalaPB) then parse and verify `protoc` & `Wire` decodes equally.

Reply:
I was also looking for getting the field tag (maybe in tag + wire type) format from `ADAPTER` instead of hard-coding here based on manually looking up the proto.

Reply:
I think that having them like this is nice

Reply:
As a comment on the `bytes` field, could you explain what you're building, specially what is missing, like there's an entry with a key but no value, viceversa.